### ISSUE TITLE
PWA not working on android device.

### ISSUE THREAD

**Issue by jiaxu09** [vercel/next.js#32131]
### What example does this report relate to?

progressive-web-app

### What version of Next.js are you using?

12

### What version of Node.js are you using?

14

### What browser are you using?

chrome

### What operating system are you using?

linux

### How are you deploying your application?

Vercel

### Describe the Bug

I run your progressive-web-app example, and deployed on vercel, pwa is working fine on PC, but not working on my android device, neither android chrome nor samsung internet.

Is there any known issue on android device?

Here is the link I deployed

https://pwa-test-red.vercel.app/

### Expected Behavior

install window should popup

### To Reproduce

Install progressive-web-app example and deployed to Vercel.

**Comment by balazsorban44** [comment]
It did for me. :thinking: What Android version are you on? What browser are you trying?

**Comment by jiaxu09** [comment]
Thanks for your reply, my phone is using android 9.

I tested on chrome, the 'add to home screen'  popup never show up, but I just found there is an 'install app' option on chrome setting, so the pwa is working on android chrome.
But on my samsung internet broswer is not working at all, could not find any 'install app' option on it, and for samsung broswer, there should be an install icon beside the search/address bar if the website is pwa supported, but the icon is not shown on the test link above.

**Comment by balazsorban44** [comment]
Do you have an example site that actually works as you expect in that browser?

**Comment by jiaxu09** [comment]
Yeah, I found this site on github build by SajjanKarn: https://covid-tracker-sajjan.vercel.app/,
this link works fine on my samsung browser.
Here is the repo:
https://github.com/SajjanKarn/covid-tracker-next
